source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -51,6 +51,10 @@ To upgrade the All-In-One setup:
 *  Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
 *  Update HA `pip3 install --upgrade homeassistant`
 
-
+Open Zwave Controller Panel:
+There is currently a bug where running ./ozwcp will complain about a missing config file. Below is how to fix it.
+*  Remove config from ```rm /srv/hass/src/open-zwave-control-panel/config```
```

Comment:
You need a new line before a list otherwise it won't render.

Also, if you plan on embedding code inline, use only 1 \`